worlds/oot_soh/Options.py

```diff
@@ -827,6 +828,14 @@ class IceTrapFillerReplacement(Range):
     default = 0
 
 
+class TricksInLogic(OptionSet):
+    """
+    Define what tricks/glitches are considered in logic
+    """
+    display_name = "Tricks in Logic"
+    valid_keys = [trick.value for trick in Tricks]
```

Comment:
```suggestion
    display_name = "Tricks in Logic"
    valid_keys = [str(trick) for trick in Tricks]
    __doc__ = ("Define what tricks/glitches are considered in logic. "
               "For more information on what each trick does, check the Ship of Harkinian "
               "Randomizer -> Seed Settings -> Tricks/Glitches settings.\n"
               "Trick names: "
               f"{", ".join(valid_keys)}")
```
This will make it so it'll display all of the trick names in the yaml

Reply:
Ooh sounds cool.